mitmproxy/master.py

```diff
@@ -76,12 +76,16 @@ def start(self):
 
     def run(self):
         self.start()
+        running = False
         try:
             while not self.should_exit.is_set():
                 # Don't choose a very small timeout in Python 2:
                 # https://github.com/mitmproxy/mitmproxy/issues/443
                 # TODO: Lower the timeout value if we move to Python 3.
                 self.tick(0.1)
+                if not running:
+                    running = True
+                    self.addons.invoke_all_with_context("running")
```

Comment:
Could we just do `self.addons.invoke_all_with_context("running")` after `self.start()` ? Or is the first `tick` doing something special?